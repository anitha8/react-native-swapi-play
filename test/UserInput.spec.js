import 'react-native';
import React from 'react';
import UserInput from '../src/components/UserInput';

import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer.create(
        <UserInput />
    ).toJSON();
    expect(tree).toMatchSnapshot();
});