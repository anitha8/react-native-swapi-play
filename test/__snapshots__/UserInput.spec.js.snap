// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Image
    source={undefined}
    style={
      Object {
        "height": 22,
        "left": 35,
        "position": "absolute",
        "top": 9,
        "width": 22,
        "zIndex": 99,
      }
    }
  />
  <TextInput
    autoCapitalize={undefined}
    autoCorrect={undefined}
    onChangeText={undefined}
    placeholder={undefined}
    placeholderTextColor="white"
    returnKeyType={undefined}
    secureTextEntry={undefined}
    style={
      Object {
        "backgroundColor": "rgba(255, 255, 255, 0.4)",
        "borderRadius": 20,
        "color": "#ffffff",
        "height": 40,
        "marginHorizontal": 20,
        "paddingLeft": 45,
        "width": 710,
      }
    }
    underlineColorAndroid="transparent"
  />
</View>
`;
